src/config.js

```diff
@@ -160,6 +160,9 @@ async function projectsConfig(url) {
       level = parseInt(level[2], 10);
     } else {
       level = level[1];
+      if (level === 'nss') {
```

Comment:
It doesn't [look](https://hg.mozilla.org/projects/nss/file/tip/.taskcluster.yml) like nss actually looks at the level that is set, but based on [#133](https://github.com/taskcluster/mozilla-taskcluster/pull/133/files#diff-7476647ff7388e1794f1358ec75b1f7dL122), I think we probably want to leave the level as `nss`.

Reply:
You are, as always, correct :)